_posts/2021-01-06-how-to-update-to-cis-13.adoc

```diff
@@ -0,0 +1,178 @@
+---
+title: How to update to CIS AWS Foundations Benchmark 1.3
+categories: Upgrades
+image: /assets/img/guides/refresh_icon.png
+excerpt: Learn how to update to the 1.3.0 version of the CIS AWS Foundations Benchmark
+tags: ["aws", "security", "compliance"]
+cloud: ["aws"]
+redirect_from: /static/guides/upgrades/how-to-update-to-cis-13/
+---
+:page-type: guide
+:page-layout: post
+
+:toc:
+:toc-placement!:
+
+// GitHub specific settings. See https://gist.github.com/dcode/0cfbf2699a1fe9b46ff04c41721dda74 for details.
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+toc::[]
+endif::[]
+
+== Intro
+
+This guide will walk you through the upgrade process from the 1.2.0 to the 1.3.0 version of the CIS AWS Foundations Benchmark.
+If your infrastructure is already compliant with the 1.2.0 version of the Benchmark, and you are looking to upgrade to 1.3.0,
+this guide is for you. If you are starting to work on compliance with this benchmark from scratch, check out our
+https://gruntwork.io/guides/compliance/how-to-achieve-cis-benchmark-compliance/[How to achieve compliance with the CIS AWS Foundations Benchmark]
+guide instead.
+
+=== What you'll learn in this guide
+
+This guide consists of two main sections:
+
+<<core_concepts>>::
+  An overview of the CIS AWS Foundations Benchmark 1.3.0 and why it is important to update your code for compatibility.
+
+<<deployment_walkthrough>>::
+  The steps you need to take to update your code to be compliant with CIS AWS 1.3.0. It includes a
+  <<compatibility_table,version compatibility table>> you can use as a reference to know which Gruntwork Repo version
+  tag is compatible with CIS AWS 1.3.0, as well as the manuals step you need to perform to achieve said compliance.
+
+
+[[core_concepts]]
+== Core Concepts
+
+=== Background
+
+The 1.3.0 version of the CIS AWS Foundations Benchmark was released in September of 2020. You can refer to the
+https://www.cisecurity.org/benchmark/amazon_web_services/[CIS website] where you can download the latest version of the
+Benchmark (as well as all the previous versions). The latest version introduces several new recommendations, and this guide
+will walk you through implementing these with help from the Gruntwork's Infrastructure as Code Library,
+so that your code becomes fully compliant with the 1.3.0 version of the framework.
```

Comment:
```suggestion
so that your infrastructure is fully compliant with version 1.3.0 of the framework.
```
Also, we are technically targeting CIS compliance for "infra" versuses "code" right? I think our solution can be used to buy compliant infrastructure, hence code, but the end customer must look after the physical controls themselves.